### ISSUE TITLE
Issues with Deprecated Protocol Syntax and Version Mismatch in Nuclei

### ISSUE THREAD

**Issue by aravindb26** [projectdiscovery/nuclei#3747]
Current Behavior:

I encountered the following issues while running Nuclei:

    Deprecated Protocol Syntax: The warning message mentions that one template (rails-debug-mode) is loaded with deprecated protocol syntax. It is advised to update the template to ensure continued support before version 2.9.5 of Nuclei.

    Templates and Nuclei-templates Version Mismatch: The output indicates that the current version of Nuclei is 2.9.4, while the current version of nuclei-templates is 9.5.0. It is recommended to use the latest versions of both Nuclei and nuclei-templates for compatibility and access to new features.

Expected Behavior:

I expected the following behavior:

    No warnings about deprecated protocol syntax in any templates.
    Consistency in the versions of Nuclei and nuclei-templates.

Steps To Reproduce:

To reproduce the issues, please follow these steps:

    Run the command nuclei -l list.txt -t ../nuclei-templates/exposures/logs/rails-debug-mode.yaml -v.
    Note the warning message about deprecated protocol syntax and the version mismatch between Nuclei and nuclei-templates.

Anything else:

I have added screenshots as proof of concept

**Comment by ehsandeep** [comment]
@aravindb26 see the pinned issue - https://github.com/projectdiscovery/nuclei/discussions/3648

**Comment by aravindb26** [comment]
@ehsandeep Yes i have updated it already, the issue is with the nuclei-template

**Comment by ehsandeep** [comment]
@aravindb26 `nuclei -l list.txt -t http/exposures/logs/rails-debug-mode.yaml` is the correct way, also you can try using `nuclei -reset` option to avoid errors.

**Comment by aravindb26** [comment]
@ehsandeep I think u r wrong mate, -t flag indicates to indicate `nuclei-templates` which accepts to analyze the vulnerabilities.

I think http is wrong `typo`

**Comment by ehsandeep** [comment]
@aravindb26, please refer to link shared in the above comment, which includes information about the directory structure update in [nuclei-templates](https://github.com/projectdiscovery/nuclei-templates) project and all the information related to the error/warning you have been experiencing.
